creator-node/src/snapbackSM.js

```diff
@@ -200,32 +199,37 @@ class SnapbackSM {
     // Issue syncs if necessary
     // For each user in the initially returned usersList,
     //  compare local primary clock value to value from secondary retrieved in bulk above
-    await Promise.all(usersList.map(
-      async (user) => {
-        let userWallet = user.wallet
-        let secondary1 = user.secondary1
-        let secondary2 = user.secondary2
-        let primaryClockValue = await this.getUserPrimaryClockValue(userWallet)
-        let secondary1ClockValue = secondaryNodeUserClockStatus[secondary1][userWallet]
-        let secondary2ClockValue = secondaryNodeUserClockStatus[secondary2][userWallet]
-        let secondary1SyncRequired = secondary1ClockValue === undefined ? true : primaryClockValue > secondary1ClockValue
-        let secondary2SyncRequired = secondary2ClockValue === undefined ? true : primaryClockValue > secondary2ClockValue
-        this.log(`${userWallet} primaryClock=${primaryClockValue}, (secondary1=${secondary1}, clock=${secondary1ClockValue} syncRequired=${secondary1SyncRequired}), (secondary2=${secondary2}, clock=${secondary2ClockValue}, syncRequired=${secondary2SyncRequired})`)
-        // Enqueue sync for secondary1 if required
-        if (secondary1SyncRequired) {
-          await this.issueSecondarySync(userWallet, secondary1, this.endpoint)
-        }
-        // Enqueue sync for secondary2 if required
-        if (secondary2SyncRequired) {
-          await this.issueSecondarySync(userWallet, secondary2, this.endpoint)
+    let numSyncsIssued = 0
+    await Promise.all(
+      usersList.map(
+        async (user) => {
+          let userWallet = user.wallet
+          let secondary1 = user.secondary1
+          let secondary2 = user.secondary2
+          let primaryClockValue = await this.getUserPrimaryClockValue(userWallet)
+          let secondary1ClockValue = secondaryNodeUserClockStatus[secondary1][userWallet]
+          let secondary2ClockValue = secondaryNodeUserClockStatus[secondary2][userWallet]
+          let secondary1SyncRequired = secondary1ClockValue === undefined ? true : primaryClockValue > secondary1ClockValue
+          let secondary2SyncRequired = secondary2ClockValue === undefined ? true : primaryClockValue > secondary2ClockValue
+          this.log(`${userWallet} primaryClock=${primaryClockValue}, (secondary1=${secondary1}, clock=${secondary1ClockValue} syncRequired=${secondary1SyncRequired}), (secondary2=${secondary2}, clock=${secondary2ClockValue}, syncRequired=${secondary2SyncRequired})`)
+          // Enqueue sync for secondary1 if required
+          if (secondary1SyncRequired) {
+            await this.issueSecondarySync(userWallet, secondary1, this.endpoint)
+            numSyncsIssued += 1
+          }
+          // Enqueue sync for secondary2 if required
+          if (secondary2SyncRequired) {
+            await this.issueSecondarySync(userWallet, secondary2, this.endpoint)
+            numSyncsIssued += 1
```

Comment:
what is purpose of `numSyncsIssues` tracking? don't see it used anywhere

Reply:
i forgot to log adding now